lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -27,12 +29,20 @@
                     this.teamsUrl = options.teamParams.teamsUrl;
                     this.languages = options.teamParams.languages;
                     this.countries = options.teamParams.countries;
-                    this.primaryButtonTitle = options.primaryButtonTitle || 'Submit';
-
-                    _.bindAll(this, 'goBackToTopic', 'createTeam');
-
-                    this.teamModel = new TeamModel({});
-                    this.teamModel.url = this.teamsUrl;
+                    this.teamsDetailUrl = options.teamParams.teamsDetailUrl;
+                    this.action = options.action;
+
+                    _.bindAll(this, 'cancelAndGoBack', 'createOrUpdateTeam');
+
+                    if (this.action === 'create') {
+                        this.teamModel = new TeamModel({});
+                        this.teamModel.url = this.teamsUrl;
+                        this.primaryButtonTitle = 'Create';
```

Comment:
Since you do have a separate i18n story (https://openedx.atlassian.net/browse/TNL-3125), I would be OK with you listing this issue in that ticket and fixing it as a follow-up PR.